docs/en/interfaces/formats/TSV/TSV.md

```diff
@@ -0,0 +1,114 @@
+During a read operation, incorrect dates and dates with times can be parsed with natural overflow or as null dates and times, without an error message.
+
+As an exception, parsing dates with times is also supported in Unix timestamp format, if it consists of exactly 10 decimal digits. The result is not time zone-dependent. The formats `YYYY-MM-DD hh:mm:ss` and `NNNNNNNNNN` are differentiated automatically.
+
+Strings are output with backslash-escaped special characters. The following escape sequences are used for output: `\b`, `\f`, `\r`, `\n`, `\t`, `\0`, `\'`, `\\`. Parsing also supports the sequences `\a`, `\v`, and `\xHH` (hex escape sequences) and any `\c` sequences, where `c` is any character (these sequences are converted to `c`). Thus, reading data supports formats where a line feed can be written as `\n` or `\`, or as a line feed. For example, the string `Hello world` with a line feed between the words instead of space can be parsed in any of the following variations:
+
+``` text
+Hello\nworld
+
+Hello\
+world
+```
+
+The second variant is supported because MySQL uses it when writing tab-separated dumps.
+
+The minimum set of characters that you need to escape when passing data in TabSeparated format: tab, line feed (LF) and backslash.
+
+Only a small set of symbols are escaped. You can easily stumble onto a string value that your terminal will ruin in output.
+
+Arrays are written as a list of comma-separated values in square brackets. Number items in the array are formatted as normally. `Date` and `DateTime` types are written in single quotes. Strings are written in single quotes with the same escaping rules as above.
+
+[NULL](/docs/en/sql-reference/syntax.md) is formatted according to setting [format_tsv_null_representation](/docs/en/operations/settings/settings-formats.md/#format_tsv_null_representation) (default value is `\N`).
+
+In input data, ENUM values can be represented as names or as ids. First, we try to match the input value to the ENUM name. If we fail and the input value is a number, we try to match this number to ENUM id.
+If input data contains only ENUM ids, it's recommended to enable the setting [input_format_tsv_enum_as_number](/docs/en/operations/settings/settings-formats.md/#input_format_tsv_enum_as_number) to optimize ENUM parsing.
+
+Each element of [Nested](/docs/en/sql-reference/data-types/nested-data-structures/index.md) structures is represented as an array.
+
+For example:
+
+``` sql
+CREATE TABLE nestedt
+(
+    `id` UInt8,
+    `aux` Nested(
+        a UInt8,
+        b String
+    )
+)
+ENGINE = TinyLog
+```
+
+``` sql
+INSERT INTO nestedt Values ( 1, [1], ['a'])
+```
+
+``` sql
+SELECT * FROM nestedt FORMAT TSV
+```
+
+``` response
+1  [1]    ['a']
+```
+
+## Example Usage
+
+## Format Settings
```

Comment:
The same here and probably for other formats with multiple variations. We can extrat format settings into a separate page and add a link in all pages with TSV* formats